packages/dictionary/README.md

```diff
@@ -1,6 +1,16 @@
-# Lectern Dictionary Meta-Schema and Utilities
+# Lectern Dictionary
 
-[![NPM Version](https://img.shields.io/npm/v/@overture-stack/lectern-dictionary?color=%23cb3837&style=for-the-badge&logo=npm)](https://www.npmjs.com/package/@overture-stack/lectern-dictionary)
+![Typescript](https://img.shields.io/badge/TypeScript-007ACC?style=for-the-badge&logo=typescript&logoColor=white)
```

Comment:
for consistency. and this being the second instance, it may be worth global-searching for the spelling across the codebase.
```suggestion
![TypeScript](https://img.shields.io/badge/TypeScript-007ACC?style=for-the-badge&logo=typescript&logoColor=white)
```

Reply:
lol fun fact: I added autocorrect for this and other few words ages ago, as I'd never want to misspell them with lowercase on purpose.
e.g. JSON, JavaScript, SnickerDoodle, BettleJuice 😆